webpack.config.js

```diff
@@ -309,4 +313,23 @@ if ( config.mode === 'development' ) {
 	config.plugins.push( new LiveReloadPlugin( { port: process.env.GUTENBERG_LIVE_RELOAD_PORT || 35729 } ) );
 }
 
+//Experimental Standalone Sandbox!
+if ( process.env.GUTENBERG_ENV === 'standalone' ) {
+	const CleanWebpackPlugin = require( 'clean-webpack-plugin' );
+	const HtmlWebpackPlugin = require( 'html-webpack-plugin' );
+	config.plugins.push( new CleanWebpackPlugin( [ 'build/standalone' ], { verbose: true } ) );
+	config.plugins.push( new HtmlWebpackPlugin( { template: './standalone/index.html' } ) );
+	config.devServer = {
+		contentBase: './build/standalone',
+	};
+	config.devtool = 'source-map';
+	config.externals = [];
+	const path = require( 'path' );
+	const alias = Object.assign( {}, config.resolve.alias, {
+		'@wordpress/core-blocks': path.resolve( __dirname, 'build/core-blocks' ),
```

Comment:
I guess you will have to do the same for `@wordpress/components`, `@wordpress/viewport` and `@wordpress/blocks` until we move them to packages :)